lib/presets/safe.es6

```diff
@@ -26,4 +26,5 @@ export default {
     removeEmptyAttributes: true,
     removeRedundantAttributes: false,
     removeComments: 'safe',
+    sortAttributesWithLists: true,
```

Comment:
Sorts values inside list-like attributes won't break anything and it is safe to enable.